.github/workflows/ci_cd.yml

```diff
@@ -324,16 +324,20 @@ jobs:
       - name: Run ingestion-server tests
         run: just ingestion_server/test-local
 
+      - name: Print ingestion-server test logs
+        run: |
+          just ingestion_server/test-logs > ingestion_server/test/ingestion_logs.txt
+          cat ingestion_server/test/ingestion_logs.txt
+
       - name: Upload ingestion test logs
         if: success() || failure()
         uses: actions/upload-artifact@v3
         with:
           name: ing_logs
           path: ingestion_server/test/ingestion_logs.txt
 
-      - name: Print ingestion test logs
-        if: success() || failure()
-        run: cat ingestion_server/test/ingestion_logs.txt
+      - name: Cleanup ingestion-server test
+        run: just ingestion_server/test-clean-dc
```

Comment:
I'm not sure, does this need an `if: success() | failure()` as well so that it always runs? @dhruvkb would know best.

Reply:
Yes I think so. I'm not entirely sure if we need the cleanup step at the end. I'm guessing since its the end of step, github actions should delete the docker containers by itself. I added it in because the CI was doing that previously. the only difference was that it was happening as part of the teardown step in pytest

Reply:
I think it's not strictly necessary to do any clean up in github actions for the reason you said, the container running it will get thrown out anyway. Maybe we can remove this step altogether?

Reply:
Removed that step